staticaddr/withdraw/manager.go

```diff
@@ -631,12 +636,23 @@ func (m *Manager) handleWithdrawal(ctx context.Context,
 						"deposits: %v", err)
 				}
 
-				// Remove the withdrawal tx from the active withdrawals
-				// to stop republishing it on block arrivals.
+				// Remove the withdrawal tx from the active
+				// withdrawals to stop republishing it on block
+				// arrivals.
 				m.mu.Lock()
 				delete(m.finalizedWithdrawalTxns, txHash)
 				m.mu.Unlock()
 
+				// Persist info about the finalized withdrawal.
+				err = m.cfg.Store.CreateWithdrawal(
```

Comment:
It'd be nicer to add the withdrawal at the time it is requested then later on set it to confirmed.